.github/workflows/release-llvm.yml

```diff
@@ -0,0 +1,231 @@
+name: Release LLVM
+
+on:
+  push:
+    tags:
+      - "llvm*"
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: create release
+        uses: softprops/action-gh-release@v2
+        with:
+          name: "LLVM binaries release: ${{ github.ref_name }}"
+          body: "This release includes binaries of LLVM, used to compile revive itself"
+          make_latest: false
+
+  build-macos:
+    strategy:
+      matrix:
+        os: [macos-14, macos-13]
+        include:
+          - os: macos-13
+            arch: x64
+          - os: macos-14
+            arch: arm64
```

Comment:
Why are we using different macOS versions for the different archs? The latest version does support Intel.

Reply:
Because that's the only available option as far as github runners go: https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

Reply:
thanks make sense